backend/core/serializers.py

```diff
@@ -600,6 +601,14 @@ class ComplianceAssessmentWriteSerializer(BaseModelSerializer):
         required=False,
         allow_null=True,
     )
+    ebios_rm_studies = serializers.SlugRelatedField(
+        many=True,
+        slug_field="id",
+        queryset=EbiosRMStudy.objects.all(),
+        required=False,
+        allow_null=True,
+        write_only=True,
+    )
```

Comment:
EBIOS RM stuff leaking on `core` should be avoided unless absolutely necessary. Since this field would only be needed in EBIOS RM contexts, what about having a serializer inheriting from this one in the `ebios_rm` app, with the additional field?